react/week1/activities.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ListItems from "./listitems";
+function Header() {
+  return <h1>Todo list</h1>;
+}
+function Activities(props) {
+  return (
+      <div>
+      <Header />
+      {props.TodoList.map((todo) => {
+        return <ListItems task={todo.task} deadline={todo.deadline} />;
+      })}
+    </div>
```

Comment:
Make sure to pay attention to proper indentation of your code. It increases readability.
```
<div>
    <Header />
    {props.TodoList.map((todo) => {
      return <ListItems task={todo.task} deadline={todo.deadline} />;
    })}
</div>
```